scripts/jenkins/log-parser/openstack-mkcloud-rules.txt

```diff
@@ -4,5 +4,10 @@ start /^Starting proposal/
 info /^\+ onadmin /
 info /^Waiting for /
 
+ok /Warning: Turning on '--gpg-auto-import-keys'/
+ok /Warning: Permanently added .* to the list of known hosts/
+warning /[Ww]arning/
+warning /WARNING/
```

Comment:
@bmwiedemann That would also match stuff like `wArNiNg`, which I didn't really want to do.  But OK whatever, it doesn't matter.